Fix heading structure in sourcemaps troubleshooting (#8142)
getsentry/sentry-docs: src/platform-includes/sourcemaps/troubleshooting/javascript.mdx

Verify your artifacts are being uploaded to Sentry. You can find them at **[Settings] > Projects > Select your project > Source Maps**.
For Sentry to de-minify your stack traces you must provide both the minified files (for example, app.min.js) and the corresponding source maps.

### Verify You Are Building Source Maps

Bundlers and tools (like `tsc`) that generate code, often require you to manually set specific options to generate source maps.

If you followed one of our tool-specific guides, verify you configured your tool to emit source maps and that the source maps contain your original source code in the `sourcesContent` field.

### Verify You Are Running A Production Build

When running JavaScript build tools (like webpack, Vite, ...) in development-mode/watch-mode, the generated code is sometimes incompatible with our source map uploading processes.

We recommend, especially when testing locally, to run a production build to verify your source maps uploading setup.

> ### Verify Your Source Files Contain Debug ID Injection Snippets

Reviewer: ```suggestion
### Verify That Source Files Contain Debug ID Injection Snippets
```

If you have the same (incorrect) results locally as you do via Sentry, double-check your source map generation configuration.

> ### Verify your source files are not too large

Reviewer: ```suggestion
### Verify That Source Files Aren't Too Large
```

For an individual artifact, Sentry accepts a max filesize of **40 MB**.

Often users hit this limit because they are transmitting source files at an interim build stage. For example, after your bundler has combined all your source files, but before minification has taken place. If possible, send the original source files.

> ### Verify artifacts are not gzipped

Reviewer: ```suggestion
### Verify That Artifacts Aren't Gzipped
```

The Sentry API currently only works with source maps and source files that are uploaded as plain text (UTF-8 encoded). If the files are uploaded in a compressed format (for example, gzip), they will be not be interpreted correctly.

If you are using `sentry-cli` to upload your artifacts, starting with version `2.4.0` you can add the `--decompress` flag to your `sourcemaps upload` or `files upload` commands.

Sometimes build scripts and plugins produce pre-compressed minified files (for example, webpack's [compression plugin](https://github.com/webpack/compression-webpack-plugin)). In these cases, you'll need to disable such plugins and perform the compression **after** the generated source maps/source files have been uploaded to Sentry.

> ### Verify workers are sharing the same volume as web (if running self-hosted Sentry via Docker)

Reviewer: ```suggestion
### Verify Workers Are Sharing Same Volume as Web (if running self-hosted Sentry via Docker)
```

Sentry does source map calculation in its workers. This means the workers need access to the files uploaded through the front end. Double check that the cron workers and web workers can read/write files from the same disk.

> ### Verify you are not using the `source-map-support` package

Reviewer: ```suggestion
### Verify You Aren't Using the `source-map-support` Package
```
